### PR TITLE
frontend/feat/#2990 Moved PP to Native

### PR DESCRIPTION
**Pull request by Vignesh-772**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Issue that needs to be resolved: Payment Page having issue with opening Multiple UPI Intent and facing issues in having PP in single instances.
Approach followed to resolve the issue: Created separate hyper instance for only PP.
Added check for debug build to fetch config.
Moved fetch assets code to purescript
### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java
Comment (witcher-shailesh) [inline Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java:1]: Can you create a separate class which will have all the PP related functions and import in bridge to use it further. Also do we want to keep it in AppBridge as of now? can we move it to just Driver's module as customer app will not have any payment flow in near future. So good to have in just driver

File: Frontend/ui-driver/index.js [hunk 1]
```diff
@@ -284,8 +288,12 @@ window["onEvent'"] = function (_event, args) {
 
 window["onEvent"] = function (jsonPayload, args, callback) { // onEvent from hyperPay
   console.log("onEvent Payload", jsonPayload);
-  if ((JSON.parse(jsonPayload)).event == "initiate_result"){
-    window.isPPInitiated = true;
+  const payload = JSON.parse(jsonPayload)
+  switch (payload.event) {
+    case "initiate_result" : window.isPPInitiated = true; break;
```
Comment (witcher-shailesh) [inline Frontend/ui-driver/index.js:293]: good to have proper indentation for better readability

File: Frontend/ui-driver/src/Flow.purs [hunk 2]
```diff
@@ -2144,6 +2144,7 @@ clearPendingDuesFlow :: Boolean -> FlowBT String Unit
 clearPendingDuesFlow showLoader = do
   void $ lift $ lift $ toggleLoader showLoader
   liftFlowBT $ runEffectFn1 initiatePP unit
+  liftFlowBT $ runEffectFn1 initiatePPSingleInstance unit
```
Comment (witcher-shailesh) [inline Frontend/ui-driver/src/Flow.purs:2147]: both initiatePP and initiatePPSingleInstance will be required to called everytime? If yes, then good to have a wrapper which call both together

**Review by witcher-shailesh — CHANGES_REQUESTED** [review]

File: Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java
Comment (witcher-shailesh) [reply Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java:1]: Create a new class, please

Comment (Vignesh-772) [reply Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java:1]: It is done.

File: Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/PaymentPage.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
+package in.juspay.mobility.common;
```
Comment (witcher-shailesh) [inline Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/PaymentPage.java:1]: Add copyright message

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**